context/app/static/js/components/Search/SelectedFilter/style.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import CancelIcon from '@material-ui/icons/CancelRounded';
+
+const StyledCancelIcon = styled(CancelIcon)`
+  height: 1.2rem;
+  margin: 0 8px;
+  color: ${(props) => props.theme.palette.primary.main};
+  cursor: pointer;
+`;
+
+const SelectedFilterDiv = styled.div`
+  background-color: white;
+  border: 1px solid ${(props) => props.theme.palette.hoverShadow.main};
+  border-radius: 8px;
+  margin: 16px 16px 0 0;
+  display: flex;
+  padding: 10px 0 10px 10px;
+  font-size: 14px;
+  line-height: 20px;
+`;
+
+const SelectedFilterName = styled.div`
```

Comment:
Can we also use a Typography component here? We would probably have to add something with `font-weight: 200` to the theme unless it will only be used here. Also worth noting that the default typography color isn't #000 elsewhere.